### PR TITLE
perf: [ANDROAPP-3189] Home screen performance

### PR DESCRIPTION
**Pull request by Balcan**
## Description
[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-3189)

## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Balcan** [comment]
Found a problem with the SDK. Wait until it is fixed to test this

File: app/src/main/java/org/dhis2/usescases/main/program/ProgramFragment.kt [hunk 1]
```diff
@@ -78,11 +83,32 @@ class ProgramFragment : FragmentGlobalAbstract(), ProgramView {
...
+        ValueAnimator.ofInt(0, 16.dp)
+            .apply {
+                duration = 700
+                startDelay = 700
+                interpolator = OvershootInterpolator()
+                addUpdateListener {
+                    val value = (it.animatedValue as Int).toFloat()
+                    gd.cornerRadii = floatArrayOf(value, value, value, value, 0f, 0f, 0f, 0f)
+                }
+            }.start()
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/main/program/ProgramFragment.kt:96]: is this animation new? should it be here????

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/main/program/ProgramFragment.kt:96]: true, it would be better in its own class

**Comment by ferdyrod** [comment]
This was tested on device and everything is working as expected. When UI test are fixed and bitrise compilation returns green check. this can be merge!

File: app/src/main/java/org/dhis2/usescases/main/program/ProgramAnimation.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            ValueAnimator.ofInt(0, 16.dp)
+                .apply {
+                    duration = 700
+                    startDelay = 700
+                    interpolator = OvershootInterpolator()
+                    addUpdateListener {
+                        val value = (it.animatedValue as Int).toFloat()
+                        gd.cornerRadii = floatArrayOf(value, value, value, value, 0f, 0f, 0f, 0f)
+                    }
+                }.start()
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/main/program/ProgramAnimation.kt:22]: PropertyValuesHolder scalex = PropertyValuesHolder.ofFloat();
            animation = ObjectAnimator.ofPropertyValuesHolder(binding.enrollmentButton, scalex, scaley);
            animation.setDuration(700);
            animation.setStartDelay(700);
            animation.setInterpolator(new OvershootInterpolator());
            animation.addUpdateListener(new ValueAnimator.AnimatorUpdateListener() {
                @Override
                public void onAnimationUpdate(ValueAnimator animation) {
                    
                }
            });
            start

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/main/program/ProgramAnimation.kt:22]: I think is straight :+1: .

**Pull request merged**